railties/test/application/test_test.rb

```diff
@@ -289,7 +321,7 @@ class UserTest < ActiveSupport::TestCase
       def assert_unsuccessful_run(name, message)
         result = run_test_file(name)
         assert_not_equal 0, $?.to_i
-        assert result.include?(message)
+        assert_includes result, message
```

Comment:
drive by cleanup: assert x.include? does not give a nice error message when it fails